# Configuracion del mapeo de ejes y botones para el paquete de ROS

# Axis mapping

axis_position_map: # Control de posicion
  x: 1
  y: 0

axis_orientation_map: # Control de orientaci√≥n
  z: 3

# Button mapping
enable_mov: 1
increment_velocity: 6
decrement_velocity: 7

# Parametros
max_displacement_in_a_second: 1 # Velocidad maxima
